### PR TITLE
Added Wishlist

### PR DESCRIPTION
**Pull request by ayush65**
Add to wishlist

### CODE REVIEW

File: src/pages/Wishlist/WishlistCard/WishlistCard.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+                <button className="btn-product-card cart-btn"
+                          onClick={() =>
+                            dispatch({
+                              type: "ADD_TO_CART",
+                              payload: { itemId: _id },
+                            })
+                          }>Add To Cart</button>
+
+              </div>
+    </div>
```
Comment (kotesh-arya) [inline src/pages/Wishlist/WishlistCard/WishlistCard.js:33]: Using prettier for formatting code would be great before merging 🙂

**Review by kotesh-arya — APPROVED** [review]
Suggested one minor thing, the rest Looks good enough for a merge 👍🏻

File: src/pages/Wishlist/WishlistCard/WishlistCard.js [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+                <h1 className="card-body cart-item-align">
+                  {itemName}
+                </h1>
+                <h1 className="card-body cart-item-align">
+                  Rs{price}
+                </h1>
+
+                <button className="btn-product-card cart-btn"
+                          onClick={() =>
+                            dispatch({
```
[the remark this answers is not in the document]
Comment (ayush65) [reply src/pages/Wishlist/WishlistCard/WishlistCard.js:26]: yea i have done the suggested changes thanks for reviewing

File: src/pages/Wishlist/WishlistCard/WishlistCard.js (hunk 1 above)
Comment (ayush65) [reply src/pages/Wishlist/WishlistCard/WishlistCard.js:32]: yea i have done the suggested changes thanks for reviewing

**Pull request merged**